contracts/src/PDPService.sol

```diff
@@ -191,9 +195,10 @@ contract PDPService {
         for (uint256 i = 0; i < rootData.length; i++) {
             addOneRoot(setId, i, rootData[i].root, rootData[i].rawSize);
```

Comment:
TODO: write comment on add function that this data is not going to be proven until the next challenge sampling event